src/bp-core/admin/bp-core-admin-schema.php

```diff
@@ -87,7 +87,9 @@ function bp_core_install( $active_components = false ) {
 	if ( ! empty( $active_components['media'] ) ) {
 		bp_core_install_media();
 		bp_core_install_document();
-		bb_core_enable_default_symlink_support();
+		if ( (bool) get_option( 'bp_media_symlink_support', false ) ) {
```

Comment:
@mahedihasannoman 
https://github.com/buddyboss/buddyboss-platform/blob/release/src/bp-core/bp-core-functions.php#L5718-L5720
Check above code. 
In `bb_core_enable_default_symlink_support` we already check this condition so i don't think it's needed

Reply:
Hi @dipskakadiya , I have removed the condition from here and updated the conditions inside the `bb_core_enable_default_symlink_support` function.